src/main/resources/public/metrics.html

```diff
@@ -0,0 +1,159 @@
+        <!-- Placed at the end of the document so the pages load faster -->
+        <script src="./js/bootstrap.min.js"></script>
+        <script src="./js/sweetalert.min.js"></script>
+        <script src="./js/Chart.bundle.min.js"></script>
+        <script src="./js/cycle.js"></script>
+        <script src="./js/datatables.min.js"></script>
+        <script type="text/javascript">
+            $(document).ready(function() {
+                $('#navbar').load('navbar.html', function() {
+                    checkIfAdmin();
+                });
+            });
+        </script>
+        <script type="text/javascript">$(document).ready(function(){  $("head").append("header.html");});</script>
+        <script type="text/javascript">getClusterName();</script>
+        <script>
+        function format ( d ) {
+            // `d` is the original data object for the row
+            let children = [];
+            for (let i = 0; i < d.ips.length; i++) {
+
+                // console.log(Object.keys(d.UserMetrics[i])[0]);
+                let ip = Object.keys(d.ips[i])[0];
+                let childObj = d.ips[i][ip];
+
+                let child =  '';
+                child += '<tr class="child-row table-condensed">';
+                child += '<td></td>';
+                child += '<td></td>';
+
+                child += '<td>' + Object.keys(d.ips[i])[0] + '</td>';
+
+                if(childObj.loginCount) {
+                    child += '<td>' + childObj.loginCount + '</td>';
+                } else {
+                    child += '<td>0</td>';
+                }
+                if(childObj.logoutCount) {
+                    child += '<td>' + childObj.logoutCount + '</td>';
+                } else {
+                    child += '<td>0</td>';
+                }
+                if(childObj.queryCount) {
+                    child += '<td>' + childObj.queryCount + '</td>';
+                } else {
+                    child += '<td>0</td>';
+                }
+                child += '</tr>';
+                children.push($(child));
+            }
+            return children
+        }
+        $(document).ready(function() {
+            $(document).on('click', 'td.details-control', function () {
+                $(this).find('.glyphicon').toggleClass('glyphicon-plus glyphicon-minus');
+                let tr = $(this).closest('tr');
+                let row = table.row( tr );
+                if ( row.child.isShown() ) {
+                    // This row is already open - close it
+                    row.child.hide();
+                    tr.removeClass('shown');
+                } else {
+                    // Open this row
+                    row.child($(format(row.data()))).show();
+                    tr.addClass('shown');
+                }
+            });
+            let table = $('#userDataTable').DataTable( {
+                "ajax": {
+                    "url": "/metrics",
```

Comment:
On this line you need to call `./metrics` in order to be proxy-friendly. Some users run NNA behind an NGINX proxy and calling `/metrics` directly won't work because need a relative path.